go.sum

```diff
@@ -212,6 +213,8 @@ github.com/smartystreets/goconvey v1.6.4/go.mod h1:syvi0/a8iFYH4r/RixwvyeAJjdLS9
 github.com/soheilhy/cmux v0.1.4/go.mod h1:IM3LyeVVIOuxMH7sFAkER9+bJ4dT7Ms6E4xg4kGIyLM=
 github.com/sonatype-nexus-community/go-sona-types v0.0.8 h1:6xb9BIC2w3y4kF/xQA25Zs6Yrl8ZqFkfH77AKPaG4RA=
 github.com/sonatype-nexus-community/go-sona-types v0.0.8/go.mod h1:uou8FGf9R5Nz1c6BfSM3v9K7g0R6faTYoxLh9Ybeht8=
```

Comment:
minor: I think `go mod tidy` would clean (remove) the old hashes for `g-s-t` if they're no longer used.

Reply:
i've run go mod tidy, it doesn't make any changes to the go.mod / go.sum file. :(

Reply:
It has some odd behavior, but `go mod tidy` is a ledger, more than a lock file. Basically just has the sha1s around to validate you got the right package. Not sure why it's keeping old stuff, but that was the problem with a `go.sum` file anyways and why we moved away from scanning it!